modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3279,10 +3280,80 @@ Response::
 ----
 --
 ====
-==== Parameters:
+==== Parameters
 
 No Parameters.
 
+=== List all running queries
+`GET /showprocesslistall`
+
+This endpoint reports the statistics of all running queries across all graphs on a TigerGraph instance: the query's request ID, start time, expiration time, and the REST endpoint's URL.
+This includes installed queries. interpreted queries, as well as built-in queries.
```

Comment:
Line 3291 through 3294 are applicable for both the APIs (as they are identical).  How can we reflect that in the document (copy the same text to both API description)?